app/services/creators/eligibilities_creator.rb

```diff
@@ -9,10 +9,14 @@ def initialize(assessment)
     end
 
     def call
-      GrossIncomeEligibilityCreator.call(@assessment)
-      DisposableIncomeEligibilityCreator.call(@assessment)
-      CapitalEligibilityCreator.call(@assessment)
-      AssessmentEligibilityCreator.call(@assessment)
+      if @assessment.assessment_type == "criminal"
```

Comment:
I think this could be `if @assessment.criminal` or similar, I feel like that's slightly neater

Reply:
Agreed.  I can add this method and make this refactor as part of the assessment ticket I'm picking up next.

Reply:
I think it's already there, but it's eminently possible I in fact put it in the wrong place...

Reply:
I realise now the "or similar" made it look like I was speculating about what the method would be called, rather I just wasn't confident about the syntax of using it